specs/player_spec.rb

```diff
@@ -0,0 +1,89 @@
+require 'minitest/autorun'
+require 'minitest/reporters'
+require 'minitest/skip_dsl'
+
+require_relative '../lib/player'
+
+# Get that nice colorized output
+Minitest::Reporters.use! Minitest::Reporters::SpecReporter.new
+
+describe 'Player' do
+  describe 'name' do
+    it 'returns the value of the @name instance variable' do
+      ada = Scrabble::Player.new("Ada")
+      ada.name.must_equal "Ada"
+    end
+  end
+
+  describe 'plays' do
+    it 'returns an Array of the words played by the player' do
+      ada = Scrabble::Player.new("Ada")
+      ada.play("apple")
+      ada.play("cat")
+      ada.plays.must_equal(["apple", "cat"])
+    end
+  end
+
+  describe 'play(word)' do
+    it 'adds the input word to the plays array' do
+      ada = Scrabble::Player.new("Ada")
+      ada.play("apple")
+      ada.plays.must_include("apple")
+    end
+
+    it 'prevents a player who has won from playing' do
+      ada = Scrabble::Player.new("Ada")
+      ada.play("apple")
+      ada.play("kale")
+      ada.play("cashew")
+      ada.play("banana")
+      ada.play("tomato")
+      ada.play("avocado")
+      ada.play("cat").must_equal(false)
+    end
+
+    it "returns the score of the word that is played if the player hasn't won" do
+      ada = Scrabble::Player.new("Ada")
+      ada.play("apple").must_equal(9)
+    end
+
+    it "if you already won it doesn't add word to array" do
+      ada = Scrabble::Player.new("Ada")
+      ada.play("apple")
+      ada.play("kale")
+      ada.play("cashew")
+      ada.play("banana")
+      ada.play("tomato")
+      ada.play("avocado")
+      ada.play("cat").must_equal false
+    end
+
+    it "updates the total score after playing word" do
+      ada = Scrabble::Player.new("Ada")
+      ada.play("beets")
+      puts ada.total_score
+    end
+  end #ends describe 'play(word)'
+
```

Comment:
You're missing tests for `Player#won?`. Here are some questions I would like to see answered by these tests:

- What if the player has not played any words at all?
- What if the player has less than 100 points?
- What if the player has exactly 100 points?
- What if the player has more than 100 points?